Formula/sshs.rb

```diff
@@ -0,0 +1,18 @@
+class Sshs < Formula
+  desc "Graphical command-line client for SSH"
+  homepage "https://github.com/quantumsheep/sshs"
+  url "https://github.com/quantumsheep/sshs/archive/refs/tags/1.3.1.tar.gz"
+  sha256 "cc6cdfe6be6f3efea29c8acc5a60ea037a88e445bced1619e7002a851d9bbe88"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "build", "VERSION=#{version}", "OUTPUT=#{bin}/sshs"
+  end
+
+  test do
+    assert_equal "sshs version #{version}", shell_output("#{bin}/sshs --version").strip
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Sadly I don't really know how to test this program differently as it starts a full UI.

Reply:
You can use Ruby's [`PTY` module](https://ruby-doc.org/stdlib-2.6.3/libdoc/pty/rdoc/PTY.html) to interact with a TUI. You can look for examples in this repository by `grep`ping for `require "pty"`.

Reply:
It should be good now